src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -378,6 +418,29 @@ private void HandleStaticIndexChange(string name, IndexDefinition definition)
             }
         }
 
+        private static void CollectPrefixesOfDocumentsToDelete(MapReduceIndex mapReduceIndex, ref HashSet<string> prefixesOfDocumentsToDelete)
```

Comment:
Why ref HashSet here? It is a class

Reply:
I like to mark it explicitly that it's modified inside the method this way